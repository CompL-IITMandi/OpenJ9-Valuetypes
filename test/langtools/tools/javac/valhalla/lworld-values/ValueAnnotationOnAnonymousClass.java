/*
 * @test /nodynamiccopyright/
 * @summary Check valueness via __value__ annotation
 * @bug 8221699
 * @compile/fail/ref=ValueAnnotationOnAnonymousClass.out -XDrawDiagnostics ValueAnnotationOnAnonymousClass.java
 */

class ValueAnnotationOnAnonymousClass {
    interface I {}
    @__inline__
    public static void main(String args []) {
        new @__inline__ I() {
        };
    }
}
