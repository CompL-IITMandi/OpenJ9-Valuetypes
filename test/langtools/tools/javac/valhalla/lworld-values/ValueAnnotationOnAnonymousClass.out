ValueAnnotationOnAnonymousClass.java:11:24: compiler.err.mod.not.allowed.here: value
ValueAnnotationOnAnonymousClass.java:12:29: compiler.err.empty.value.not.yet
2 errors
